docs/docs/setting-up-spring-boot-service/configuring-axelix-starter/configuring-axelix-starter.mdx

```
@@ -1 +1,262 @@
-# Configuring Axelix Starter
+import Tabs from '@theme/Tabs';
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)